tests/mutaterows_test.go

```diff
@@ -336,17 +336,18 @@ func TestMutateRows_Retry_ExponentialBackoff(t *testing.T) {
 
 	// 4b. Log the retry delays
 	origReq := <-recorder
-	firstRetry := <-recorder
-	secondRetry := <-recorder
-	thirdRetry := <-recorder
 
-	firstDelay := int(firstRetry.ts.UnixMilli() - origReq.ts.UnixMilli())
-	secondDelay := int(secondRetry.ts.UnixMilli() - firstRetry.ts.UnixMilli())
-	thirdDelay := int(thirdRetry.ts.UnixMilli() - secondRetry.ts.UnixMilli())
-
-	// Different clients may have different behaviors, we log the delays for informational purpose.
-	// Example: For the first retry delay, C++ client uses 100ms but Java client uses 10ms.
-	t.Logf("The three retry delays are: %dms, %dms, %dms", firstDelay, secondDelay, thirdDelay)
+	for n := 1; n < numRPCs; n += 1 {
```

Comment:
Maybe in the end we should assert that n == 3? so it doesn't hang and we can verify the client retried 3 times?

Reply:
true :) ok